pegs_common_utils.R

```diff
@@ -258,3 +258,70 @@ prepare_pegs_phenotype <- function(pegs_data, phenotype) {
 
   return(pegs_pheno_prepd)
 }
+
+#' PEGS data converter
+#'
+#' convert PEGS text table entries to NA and proper R types;
+#'
+#' this function works for text tables and meta-data from PEGS Freeze v2,
+#' loaded from
+#'
+#' - /ddn/gs1/project/controlled/PEGS/Data_Freezes/freeze_v2/Surveys
+#'   * Health_and_Exposure/healthexposure_*_v2.* # epr.he and epr.he.meta
+#'   * Exposome/exposomea_*_v2.*                 # exposome a
+#'   * Exposome/exposomea_*_v2.*                 # exposome b
+#'
+#' @param dat one of the PEGS text tables (i.e., epr.he)
+#' @param met corresponding meta-data (i.e., epr.he.meta)
+#' @return data frame with NA and proper column classes.
+cvt <- function(dat, met, quiet=TRUE)
+{
+    ## code for NA
+    NAS <- c(".M", ".S", ".N", "-444444", "-555555", "-666666", "-777777", "-888888", "-999999")
```

Comment:
Use the global PEGS_SP_CODES

Reply:
if **PEGS_SP_CODES** is only used in the converter function, we should keep it in the function.
Are you using it somewhere other than **PEGS_common.R** (i.e., scripts not yet added into the github)?